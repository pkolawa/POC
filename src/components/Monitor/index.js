import React from "react";
import Typography from "@material-ui/core/Typography";

class Monitor extends React.Component {
  render() {
    return (
      <div>
        <Typography align="center" variant="h2" color="secondary">
          Monitor Page
        </Typography>
      </div>
    );
  }
}

export { Monitor as default }