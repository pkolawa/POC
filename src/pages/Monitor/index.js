import MonitoringSystemContainer from "../../containers/MonitoringSystemContainer";

export default MonitoringSystemContainer;