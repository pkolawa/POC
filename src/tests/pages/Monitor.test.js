import React from "react";
import { shallow } from "enzyme";
import Monitor from "../../pages/Monitor";

test("Should Render Configuration Page", () => {
  const wrapper = shallow(<Monitor/>);
  expect(wrapper).toMatchSnapshot();
});