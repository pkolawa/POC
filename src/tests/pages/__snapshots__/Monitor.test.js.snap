// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Configuration Page 1`] = `
<Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Monitor
    </title>
    <meta
      content="Monitor"
      name="description"
    />
  </HelmetWrapper>
  <Connect(WithStyles(MonitoringSystemContainer)) />
</Fragment>
`;
